src/generator/index.ts

```diff
@@ -80,29 +83,30 @@ export class Generator {
       return JSON.parse(process.env.TEST_QUESTIONS);
     }
 
-    const { onCloud } = await prompt<{ onCloud: string }>({
-      type: 'confirm',
-      name: 'onCloud',
-      initial: 'y',
-      message: 'Do you use Elastic Cloud',
-    });
     const url = await new Input({
-      header: onCloud
-        ? yellow(
-            'Get cloud.id from your deployment https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html'
-          )
-        : '',
-      message: onCloud
-        ? 'What is your cloud.id'
-        : 'What is the url of your Kibana instance',
       name: 'url',
+      message: 'Enter Elastic Kibana URL or Cloud ID',
       required: true,
-      result(value) {
-        return onCloud ? cloudIDToKibanaURL(value) : value;
+      validate(value) {
+        if (!IS_URL.test(value)) {
+          value = cloudIDToKibanaURL(value);
```

Comment:
Can we include this statement in the try/catch statement? `cloudIDToKibanaURL` is likely to throw if gibberish is passed

Reply:
Good point.

Reply:
Same train of thought here, the UX is kind of bad in the case you enter garbage. For instance, this is what I see:

```
➜ npx @elastic/synthetics init leafeon
> Initializing Synthetics project in 'leafeon'
? Enter Elastic Kibana URL or Cloud ID › garbage  

❯ Invalid URL or Cloud ID
```

Once this happens, all of my keystrokes are just met with a beep and I can't try again. I had to kill the process, which is actually more annoying than if it just blew up and crashed on its own. How feasible is it to let us try again after seeing this error?

Reply:
Thats weird, I was able to use backspace/delete and re-enter again to go to the next step.

Reply:
Please try again and let me know how it goes, Happy to remove the validation later if you still get hit by this.

Reply:
Just an error on my part, everything is good here!